game/unicode.hh

```diff
@@ -2,30 +2,57 @@
 
 #include <iostream>
 #include <map>
+#include <memory>
+#include <mutex>
 #include <string>
 #include <unicode/errorcode.h>
 #include <unicode/tblcoll.h>
 #include <unicode/uclean.h>
+#include <unicode/ucnv.h>
 #include <unicode/ucsdet.h>
 
 typedef std::map<std::string, std::string> songMetadata;
 
+class UnicodeString;
+
+struct Converter {
+	/// Constructor/Destructor
+	Converter(std::string const& codepage);
+	Converter(Converter&& c) noexcept;
+	Converter(Converter& c) = delete;
+	Converter(Converter const& c) = delete;	
+	~Converter() {};
```

Comment:
Not needed cause the compiler will create it automatically.

Reply:
Which one, the destructor?

Reply:
Yes, there's always a default dtor if none is defined.